server/src/controllers/employer_contact/deleteEmployerContact.js

```diff
@@ -0,0 +1,33 @@
+const logger = require("pino")();
+const EmployerContact = require("../../models/employer_contact.model");
+
+const deleteEmployerContactRequestHandler = async (req, res) => {
+  try {
+    let employer_contact = req.params.employer_contact_id;
+
+    employer_contact = await EmployerContact.findOne({
+      where: { id: employer_contact },
+    });
+    if (employer_contact) {
+      await employer_contact.destroy();
+      return res.status(200).json({
+        status: "success",
+        message: "Employer Contact Successfully deleted",
+        data: {},
```

Comment:
for when the user deletes an employer contact, would it be possible to also return the object that was deleted in case the frontend would like to have access to what was deleted